client_delivery/shared/src/commonMain/kotlin/presentation/login/composable/WrongPermissionBottomSheet.kt

```diff
@@ -0,0 +1,62 @@
+package presentation.login.composable
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+import com.beepbeep.designSystem.ui.composable.BpButton
+import com.beepbeep.designSystem.ui.composable.BpTransparentButton
+import com.beepbeep.designSystem.ui.theme.Theme
+import org.jetbrains.compose.resources.ExperimentalResourceApi
+import org.jetbrains.compose.resources.painterResource
+import presentation.login.LoginScreenInteractionListener
+import resources.Resources
+
+@OptIn(ExperimentalResourceApi::class, ExperimentalMaterial3Api::class)
+@Composable
+fun WrongPermissionBottomSheet(listener: LoginScreenInteractionListener,
+                               modifier: Modifier = Modifier) {
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = modifier.wrapContentHeight()
+            .padding(horizontal = Theme.dimens.space16, vertical = Theme.dimens.space24)
+    ) {
+        Icon(
+            painter = painterResource(Resources.images.errorIcon),
+            tint = Theme.colors.primary,
+            contentDescription = null,
```

Comment:
Don't make assign any content description to null , you should assign a string value from Resourses

Reply:
thanks for your review, i thought that this is some style you agree to use so i follow the previous style